trainer/im_utils.py

```diff
@@ -219,7 +220,7 @@ def load_random(train_annot_dirs, train_seg_dirs, dataset_dir, _):
 
         # it's possible the image has a different extenstion
         # so use glob to get it
-        fname_no_ext = fname.replace('.nii.gz', '').replace('.nrrd', '')
+        fname_no_ext = fname.replace('.nii', '').replace('.nii.gz', '').replace('.nrrd', '')
```

Comment:
I think this might cause a bug. .nii.gz would be replaced with .gz

Reply:
I think we should add a 'set_ext' function that handles this. See previous comment.